File: Backend/TrackrAPI/TrackrAPI/Services/Inventario/DomicilioService.cs
using TrackrAPI.Models;
using TrackrAPI.Repositorys.Inventario;

namespace TrackrAPI.Services.Inventario
{
    public class DomicilioService
    {
        private IDomicilioRepository domicilioRepository;
        public DomicilioService(
            IDomicilioRepository domicilioRepository
            )
        { 
            this.domicilioRepository = domicilioRepository;
        }

        public Domicilio Consultar(int idDomicilio)
        {
            return domicilioRepository.Consultar(idDomicilio);
        }
    }
}

File: Frontend/trackr-administrador/src/app/shared/http/inventario/domicilio.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { Domicilio } from '@models/seguridad/domicilio';

@Injectable({
    providedIn: 'root'
})
export class DomicilioService {
    private dataUrl = 'domicilio/';

    constructor(public http: HttpClient) {}

    // TODO: Change ExpedienteWrapper location
    public consultar(idDomicilio: number) {
        return this.http.get<Domicilio>(this.dataUrl + `consultar/${idDomicilio}`);
    }
}